README.md

```diff
@@ -106,7 +114,9 @@ It has convenient hyperlinks that navigate you around the text.<br>
  - uses [CMYK](https://en.wikipedia.org/wiki/CMYK_color_model) (experimental) 🎨
 
 all of which to be print friendly.
-Best served for those who would like to have it on paper 🤞
+Best served for those who would like to have it on paper 🤞<br>
+🤑 The economy printable build has **no art backgrounds**. Other than that, it's the same as the regular printable build.
+This is for people who want to save their toners or don't have printers that can print edge to edge where the white borders around the art backgrounds don't look that good.
```

Comment:
we should make printable build (background or not) to use crop and not a4 directly; but that's out of scope for this change